### PR TITLE
custom component creation issue fix

### PR DESCRIPTION
**Pull request by khus29**

### CODE REVIEW

File: pages/js-templates/template.js.hbs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (vinodloha) [inline pages/js-templates/template.js.hbs:1]: Why ? we have separate templates we can make use of existing components to be created at /template/*

File: react-component/index.js [hunk 2]
```diff
@@ -35,18 +35,19 @@ export default {
       when: (data) => data.folder === 'custom',
```
Comment (vinodloha) [inline react-component/index.js:35]: I think we should change this option data.folder === 'custom', to may be 

data.folder === 'custom-path',

File: pages/js-templates/template.js.hbs (hunk 1 above)
Comment (khus29) [reply pages/js-templates/template.js.hbs]: Done. Referring to existing component templates.

File: react-component/index.js (hunk 2 above)
Comment (khus29) [reply react-component/index.js]: Done

**Review by vinodloha — APPROVED** [review]

**Pull request merged**